scala3-scala2-library-tasty-tests/test/Scala3Scala2LibraryTASYyTest.scala

```diff
@@ -0,0 +1,206 @@
+package dotty.tools.dotc
+
+import org.junit.Test
+import org.junit.Ignore
+import org.junit.Assert._
+
+import dotty.tools.io._
+import dotty.tools.dotc.util.ClasspathFromClassloader
+
+import scala.quoted._
+
+import java.io.File.pathSeparator
+
+class Scala3Scala2LibraryTASYyTest:
+
+  import Scala3Scala2LibraryTASYyTest._
+
+  /** Test that we can load trees from TASTy */
+  @Test def testTastyInspector: Unit =
+    loadWithTastyInspector(loadBlacklisted)
+
+  /** Test that we can load and compile trees from TASTy */
+  @Test def testFromTasty: Unit =
+    compileFromTasty(loadBlacklisted.union(compileBlacklisted))
+
+  @Ignore
+  @Test def testWhiteListFromTasty: Unit =
```

Comment:
What do we need a whitelist for?

Reply:
It was a simple way to test a subset of files. Now that I have minimized everything I can remove it.

Reply:
On second thought, I prefer to keep in case a change breaks the compilation of the library. This would help to minimize the failing combination of files faster.

Reply:
That's fine but then please add a comment on top of the test explaining that